releases/release-1.11/release_notes_draft.md

```diff
@@ -428,12 +428,11 @@ If excludeMasterFromStandardLB is not set, it will be default to true, which mea
 * Create a new `dryRun` query parameter for mutating endpoints. If the parameter is set, then the query will be rejected, as the feature is not implemented yet. This will allow forward compatibility with future clients; otherwise, future clients talking with older apiservers might end up modifying a resource even if they include the `dryRun` query parameter. (#63557, @apelisse)
 * Search standard KubeConfig file locations when using `kubeadm token` without `--kubeconfig`. (#62850, @neolit123)
 *   Include the list of security groups when failing with the errors that more then one is tagged (#58874, @sorenmat)
-* Allow "required" to be used at the CRD OpenAPI validation schema when the /status subresource is enabled. (#63533, @sttts)
+* _[obsolete]_ Allow "required" to be used at the CRD OpenAPI validation schema when the /status subresource is enabled. (#63533, @sttts)
 * try to read openstack auth config from client config and fall back to read from the environment variables if not available (#60200, @dixudx)
 * Services can listen on same host ports on different interfaces with --nodeport-addresses specified (#62003, @m1093782566)
 * Adding initial Korean translation for kubectl (#62040, @ianychoi)
 * Report node DNS info with --node-ip flag (#63170, @micahhausler)
-* CustomResourceDefinitions Status subresource now supports GET and PATCH (#63619, @roycaihw)
```

Comment:
This was a duplicate. It's already present in sig-api-machinery pending topics. I have edited this one above.